src/ServicePulse.Host/vue/src/composables/serviceServiceControl.js

```diff
@@ -151,17 +151,20 @@ export function useServiceControlStats() {
   const failedHeartBeatsResult = getFailedHeartBeatsCount();
   const failedMessagesResult = getFailedMessagesCount();
   const failedCustomChecksResult = getFailedCustomChecksCount();
+  const archivedMessagesResult = getArchivedMessagesCount();
 
   return Promise.all([
     failedHeartBeatsResult,
     failedMessagesResult,
     failedCustomChecksResult,
+    archivedMessagesResult,
   ])
-    .then(([failedHB, failedM, failedCC]) => {
+    .then(([failedHB, failedM, failedCC, archivedM]) => {
```

Comment:
I know these weren't introduced in your PR, but can we get these better names while we're here?

Reply:
No problem.  I made the update [here](https://github.com/Particular/ServicePulse/pull/1372/files#diff-8fb1814bd80cda2e7aeeb1a7745ef24b178bbaf4e4e8a8da0f566145fcc9d8c8R164-R173)